npm/design-system/src/components/searchInput/SearchInput.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { FormEvent, MutableRefObject, useCallback } from 'react'
```

Comment:
Can't we do `import React, { otherStuff } from 'react'`?

Reply:
Yes. Bad habit.